### PR TITLE
Warn against use of 17.3 due to altlogrotate bug

### PR DESCRIPTION
**Pull request by jrha**
i.e. quattor/configuration-modules-core#1141

### CODE REVIEW

File: _posts/2017-06-15-announcing-quattor-17.3.0.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ category: news
 author: James Adams
 ---
 
+<div class="alert alert-danger" role="alert">
+    <p><b>Warning</b>: This release contains a [serious bug](https://github.com/quattor/configuration-modules-core/issues/1141) that can break logrotate configuration. We recommend that you do not deploy it unless you are certain you will not be affected.</p>
```
Comment (stdweird) [inline _posts/2017-06-15-announcing-quattor-17.3.0.md]: well, it's the 3rd serious bug (see also https://github.com/quattor/ncm-cdispd/pull/52 and https://github.com/quattor/CCM/pull/174).
will you replane the 17.3.0 ncm-altlogrotate with the previopus release in the repo?

Comment (jrha) [inline _posts/2017-06-15-announcing-quattor-17.3.0.md]: pardon?

Comment (stdweird) [inline _posts/2017-06-15-announcing-quattor-17.3.0.md]: swap ncm-altlogrotate 17.3.0 with copy of 17.2.0? (either in yum repo or in template library, or both)

Comment (jrha) [inline _posts/2017-06-15-announcing-quattor-17.3.0.md]: OIC, not a bad idea.

**Pull request merged**